src/DIRAC/FrameworkSystem/ConfigTemplate.cfg

```diff
@@ -1,3 +1,12 @@
+APIs
+{
+  Auth
```

Comment:
You should use the `##BEGIN Auth` followed by `##END` and use it.

Reply:
done